
libvorbis was badly installed (multiple copies of .so in /usr/lib/ instead of
symlinks). 
This patch updates to the latest Buildroot state to correct that.
This patch will have to be removed when updating to latest Buildroot release.

Index: buildroot/package/audio/libvorbis/libvorbis.mk
===================================================================
--- buildroot.org/package/audio/libvorbis/libvorbis.mk	2009-06-03 10:36:05.000000000 +0200
+++ buildroot/package/audio/libvorbis/libvorbis.mk	2009-06-03 11:01:31.000000000 +0200
@@ -6,57 +6,17 @@
 LIBVORBIS_VERSION:=1.2.0
 LIBVORBIS_SOURCE:=libvorbis-$(LIBVORBIS_VERSION).tar.gz
 LIBVORBIS_SITE:=http://downloads.xiph.org/releases/vorbis/$(LIBVORBIS-SOURCE)
-LIBVORBIS_DIR:=$(BUILD_DIR)/libvorbis-$(LIBVORBIS_VERSION)
-LIBVORBIS_BINARY:=libvorbis
-LIBVORBIS_TARGET_BINARY:=usr/bin/libvorbis
-LIBVORBIS_CAT:=$(ZCAT)
 
-$(DL_DIR)/$(LIBVORBIS_SOURCE):
-	$(WGET) -P $(DL_DIR) $(LIBVORBIS_SITE)/$(LIBVORBIS_SOURCE)
+LIBVORBIS_AUTORECONF = NO
+LIBVORBIS_INSTALL_STAGING = YES
+LIBVORBIS_INSTALL_TARGET = YES
 
-$(LIBVORBIS_DIR)/.source: $(DL_DIR)/$(LIBVORBIS_SOURCE)
-	$(LIBVORBIS_CAT) $(DL_DIR)/$(LIBVORBIS_SOURCE) | tar -C $(BUILD_DIR) $(TAR_OPTIONS) -
-	touch $@
-
-$(LIBVORBIS_DIR)/.configured: $(LIBVORBIS_DIR)/.source
-	(cd $(LIBVORBIS_DIR); rm -rf config.cache; \
-		$(TARGET_CONFIGURE_ARGS) \
-		$(TARGET_CONFIGURE_OPTS) \
-		CFLAGS="$(TARGET_CFLAGS)" \
-		./configure \
-		--target=$(GNU_TARGET_NAME) \
-		--host=$(GNU_TARGET_NAME) \
-		--build=$(GNU_HOST_NAME) \
-		--prefix=/usr \
-		--sysconfdir=/etc \
-		--enable-shared \
-		--enable-static \
-		--disable-oggtest \
-		$(DISABLE_NLS) \
-	)
-	touch $@
-
-$(LIBVORBIS_DIR)/.libs: $(LIBVORBIS_DIR)/.configured
-	$(MAKE) -C $(@D)
-	touch $@
-
-$(STAGING_DIR)/usr/lib/libvorbis.so: $(LIBVORBIS_DIR)/.libs
-	$(MAKE) DESTDIR=$(STAGING_DIR) -C $(LIBVORBIS_DIR) install
-
-$(TARGET_DIR)/usr/lib/libvorbis.so: $(STAGING_DIR)/usr/lib/libvorbis.so
-	$(INSTALL) -D $(STAGING_DIR)/usr/lib/libvorbis*.so* $(TARGET_DIR)/usr/lib
-	$(STRIPCMD) $(STRIP_STRIP_UNNEEDED) $(TARGET_DIR)/usr/lib/libvorbis*.so*
-
-libvorbis: uclibc pkgconfig libogg $(TARGET_DIR)/usr/lib/libvorbis.so
+LIBVORBIS_CONF_OPT = --disable-oggtest
 
-libvorbis-source: $(DL_DIR)/$(LIBVORBIS_SOURCE)
+LIBVORBIS_DEPENDENCIES = uclibc pkgconfig libogg
 
-libvorbis-clean:
-	$(MAKE) prefix=$(STAGING_DIR)/usr -C $(LIBVORBIS_DIR) uninstall
-	-$(MAKE) -C $(LIBVORBIS_DIR) clean
+$(eval $(call AUTOTARGETS,package/multimedia,libvorbis))
 
-libvorbis-dirclean:
-	rm -rf $(LIBVORBIS_DIR)
 
 ############################################################
 #
